Update theme.md - creation of wwwroot folder was missing (#15265)
OrchardCMS/OrchardCore: src/docs/getting-started/theme.md

- In Visual Studio Code or CLI, execute the command `dotnet sln add MyTheme.OrchardCore.csproj` to add project to solution. Then, go to `OrchardCore.Cms.Web` folder and execute the command `dotnet add OrchardCore.Cms.Web.csproj reference ../OrchardCore.Themes/MyTheme.OrchardCore.csproj`
- Set the main Orchard Core CMS Web application as the startup project.

> A thumbnail can also be used by the Admin UI by adding a `wwwroot\Theme.png` folder/file in your theme project root folder.

Reviewer: ```suggestion
 - The Admin UI can utilize a thumbnail by incorporating a `wwwroot\Theme.png` file within the root folder of the theme project.
```

> Since `MyTheme.OrchardCore` is a library project and doesn't include a `wwwroot` folder by default, you'll need to create one. To add a thumbnail to `MyTheme.OrchardCore` template, you can place a `Theme.png` file inside the `wwwroot` folder.

Reviewer 2: ```suggestion
A thumbnail can also be used by the Admin UI by adding a `wwwroot\Theme.png` folder/file in your theme project root folder.
```

Author: @Skrypt thank you

Reviewer 2: Ok but you decided to not commit my suggestion?

Author: @Skrypt I think I clicked the wrong button 😅
